cmd2/constants.py

```diff
@@ -14,6 +14,7 @@
 MULTILINE_TERMINATOR = ';'
 
 LINE_FEED = '\n'
+HORIZONTAL_ELLIPSIS = '\N{HORIZONTAL ELLIPSIS}'
```

Comment:
Can you explain how this works?  Like is it filling in an ellipsis `...`?  If so where is that coming from?

Reply:
It's an escape sequence for the one-character ellipsis. (…) It's easier type than the real thing, but I will alter the code to make it clearer.